bugbot/crash/analyzer.py

```diff
@@ -477,6 +477,41 @@ def is_near_allocator_related_crash(self) -> bool:
         """
         return self.is_near_allocator_crash or self.is_potential_near_allocator_crash
 
+    @property
+    def is_content_crash(self) -> bool:
+        """Whether the crash is related to content process."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "content":
+                return row["count"] > 0
+        return False
+
+    @property
+    def is_gpu_crash(self) -> bool:
+        """Whether the crash is related to GPU."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "gpu":
+                return row["count"] > 0
+        return False
+
+    @property
+    def is_parent_crash(self) -> bool:
+        """Whether the crash is related to the parent process."""
+        for row in self.signature["facets"]["process_type"]:
+            if row["term"].lower() == "parent":
+                return row["count"] > 0
+        return False
+
+    @property
+    def crash_kind(self) -> str:
+        """The kind of the crash based on the crashing process type."""
+        if self.is_plugin_crash:
+            return "Plugin"
+        if self.is_parent_crash:
+            return "Parent"
+        if self.is_content_crash:
+            return "Content"
+        return "Browser Crash"
```

Comment:
I think the default case should be more like "Other". To me, in the context of crash stats "Browser crash" usually means "crash in the parent process". Also the other ones are process types while this is sort of like a crash type.

Reply:
The goal here is to show crash type instead of process type. Originally, it was showing either "Browser Crash" or "Plugin Crash". I think growing in this direction will be more useful.

Reply:
Ok, that makes sense. As I said above, I personally don't find the notion of a "crash type" useful and I just want the process type. Maybe somebody finds the "crash type" thing useful. I don't know.